parsers/UA.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import arrow
+import dateutil
+import requests
+
+MAP_GENERATION = {
+  'aes': 'nuclear',
+  'tec': 'other', # fossil 
+  'tes': 'coal',
+  'chpp': 'gas',
+  'hpp': 'hydro',
+  'vde': 'wind',
+  'gesgaes' : 'gesgaes', # hydro pumped storage
+  'consumptiongaespump' :'consumptiongaespump' #hydro run of river and poundage

```

Comment:
the pumped storage looks to report both ways: positive when generating, negative when it is pumped back up. 
I combined the two because there is only one hydro generation; the sum of both is placed under hydro. Let me know if it should go different.